tests/onnx/test_e2e_ptq.py

```diff
@@ -51,44 +57,25 @@
 TASKS = ["classification", "object_detection_segmentation"]
 ALL_MODELS = [(task, os.path.splitext(model)[0]) for task in TASKS for model in
               os.listdir(BENCHMARKING_DIR / task / "onnx_models_configs")]
-
+# Default E2E Scope of Models
 E2E_MODELS = [(task_type, model_name) for task_type, model_name in ALL_MODELS if model_name in
               ['densenet-12', 'mobilenetv2-12', 'resnet50-v2-7', 'shufflenet-v2-12', 'squeezenet1.0-12',
                'efficientnet-lite4-11', 'inception-v1-12',
-               'ssd-12', 'yolov3-12', 'yolov4', 'ResNet101-DUC-12', 'FasterRCNN-12', 'retinanet-9']]
-
-XFAIL_MODELS = {}
-
-XFAIL_QUANTIZED_MODELS = {
+               'ssd-12', 'yolov3-12', 'yolov4', 'ResNet101-DUC-12', 'FasterRCNN-12', 'MaskRCNN-12', 'retinanet-9']]
+# Fail Model and Reason of Failure
+FAIL_MODELS = {
+    # model name: reason of skipping
+    'MaskRCNN-12': 'ticket 102051',
+    'yolov4': 'ticket 99211',
+    'yolov3-12': 'ticket 99211'
 }
 
-# TODO(vshampor): Somehow installing onnxruntime-openvino does not install the OV package in the way
-#  that we are used to, and accuracy checker invocations cannot find the dataset_definitions.yml
-#  file using the regular path pointing to site-packages, hence the need for using a copy of
-#  a hopefully relevant dataset_definitions.yml taken from the tests dir. Should investigate
-#  if onnxruntime-openvino actually has a relevant dataset_definitions.yml file somewhere within its own
-#  site-packages directory.
-DATASET_DEFINITIONS_PATH_ONNX = BENCHMARKING_DIR / 'dataset_definitions.yml'
-
-OV_EP_COL_NAME = "OpenVINOExecutionProvider"
-OV_COL_NAME = "OpenVINO"
-CPU_EP_COL_NAME = "CPUExecutionProvider"
-REPORT_NAME = 'report.html'
-
-
-def check_xfail(model_name):
-    if model_name in XFAIL_MODELS:
-        pytest.xfail("ONNXRuntime-OVEP cannot execute the reference model")
-
-
-def check_quantized_xfail(model_name):
-    if model_name in XFAIL_QUANTIZED_MODELS:
-        pytest.xfail("ONNXRuntime-OVEP cannot execute the quantized model")
-
 
 def check_skip_model(model_name: str, model_names_to_test: Optional[List[str]]):
     if model_names_to_test is not None and model_name not in model_names_to_test:
         pytest.skip(f'The model {model_name} is skipped, because it was not included in --model-names.')
+    if model_name in FAIL_MODELS:
+        pytest.skip(f'The model {model_name} is skipped, {FAIL_MODELS[model_name]}')
```

Comment:
`xfail`. `xfail` is used when we actually expect the model to work after an external issue has been resolved.

Reply:
Add xfail for failed models